src/components/taskDetails/index.tsx

```diff
@@ -208,6 +248,36 @@ const TaskDetails: FC<Props> = ({ url, taskID }) => {
                                     />
                                 </div>
                             </TaskContainer>
+                            <TaskContainer
+                                title="Task DependsOn"
+                                hasImg={false}
+                            >
+                                <ol
+                                    className={
+                                        classNames[
+                                            'task_dependency_list_container'
+                                        ]
+                                    }
+                                >
+                                    {taskTitle.map((title, index) => (
```

Comment:
can we have this ?
{taskTitle.length === 0 ? (
        <p>No Dependency</p>
    ) : (
        taskTitle.map((title, index) => (
            <Link href={/tasks/${id[index]}} key={index}>
                <li onClick={() => navigateToTask(id[index])}>{title}</li>
            </Link>
        ))
    )}

Reply:
I have also similar doubt...now @khushi818 corrected...Thanku @khushi818 
cc: @kotesh-arya you check once

Reply:
Done